docs/README.md

```diff
@@ -61,16 +66,14 @@ For example, to disable on staging:
 
 ### Editor Stack Traces
 
-When viewing a stack trace of a PHP warning or error, the developer tools can turn these into clickable links that open in your editor. To enable this, define `QM_LOCAL_EDITOR` like this:
+Stack traces within the developer tools panel can be made into clickable links that open in your editor. Database queries will also be clickable so you can jump directly to the function that made them.
 
-```php
-define( 'QM_LOCAL_EDITOR', 'phpstorm' );
-```
+To activate clickable stack traces:
 
-Valid values include:
+1. Open the developer tools panel
+2. Click on the gear icon in the top right to access the settings screen
+3. Select your editor from the drop down
+4. Click the "Set editor cookie" button
+5. Refresh the page
 
- - `phpstorm`
- - `vscode`
- - `atom`
- - `sublime`
- - `netbeans`
+![Screenshot of the developer tools panel](./assets/qm-settings.png)
```

Comment:
We probably don't need the same screenshot in both spots?

Reply:
Yeah perhaps, or we should highlight the relevant part in each image